.gitignore

```diff
@@ -1,3 +1,26 @@
-_build/
-deps/
-doc/
+# The directory Mix will write compiled artifacts to.
```

Comment:
No problem with adding a leading / to all directories, but the other changes should be reverted.

For example I have never seen a /tmp/ here so it should not be ignored. The comment lines are imho somewhat unhelpful and with the generic library setup there probably won't be anything new in the foreseeable future.

Reply:
The previous comment still applies, there should be no changes to this file except maybe the leading slash.